controller/contacts.js

```diff
@@ -0,0 +1,94 @@
+const service = require("../service/contacts.js");
+
+const {
+  postValidator,
+  putValidator,
+  patchValidator,
+} = require("../utils/validationContacts");
+
+const getContacts = async (req, res, next) => {
+  const contacts = await service.listContacts();
+  res.status(200).json(contacts);
+};
+
+const getContactById = async (req, res, next) => {
+  const { contactId } = req.params;
+  const contact = await service.getContactById(contactId);
+  if (contact) {
+    return res.status(200).send(contact);
+  }
+  if (!contact) {
+    return res.status(404).json({ message: "Not found" });
+  }
```

Comment:
definitywnie przyklad na `if..else`, ktory mozesz dalej zamienic na ternary operator:
```js
return contact
? res.status(200).send(contact)
: res.status(404).json({ message: "Not found" });
```

nie uzywaj `send` (!!!), bo to wysyla string i w dodatku jeszcze sprawiasz, ze API odpowiada roznymi formatami. zmien `send` na `json` i zawsze odsylaj JSON.